tests/test_cli.py

```diff
@@ -416,15 +417,16 @@ def test_invalid_order_no_input_file(script_runner):
 
 def test_zenodo_valid_doi_repository_wrong_geopackage_extension(script_runner):
     with pytest.warns(ResourceWarning):
-        ret = script_runner.run('geoextent', '-b', '-t', '--output', 'wrong_extension.abc',
+        with tempfile.NamedTemporaryFile(suffix=".abc") as tmp:
+            ret = script_runner.run('geoextent', '-b', '-t', '--output', tmp.name,
                                 'https://doi.org/10.5281/zenodo.820562'
-                                )
+                                    )
     assert ret.success, "process should return success"
 
 
 def test_export_absolute_path(script_runner):
     with tempfile.TemporaryDirectory() as tmp:
-        out_path = tmp + "geoextent_output.gpkg"
+        out_path = tmp + "/geoextent_output.gpkg"
```

Comment:
This might break on some systems with different path formatting, please only use `os.path.join()`.

Reply:
Fixed !